onnxruntime/hosting/server/beast_http.h

```diff
@@ -19,10 +22,12 @@ namespace net = boost::asio;       // from <boost/asio.hpp>
 using tcp = boost::asio::ip::tcp;  // from <boost/asio/ip/tcp.hpp>
```

Comment:
Consider put the using into the namespace?

Reply:
Why? I feel like this is cleaner.

Reply:
OK I did it.